packages/storybook/stories/components/progress-bar/migration.from.17.x.mdx

```diff
@@ -0,0 +1,14 @@
+import { Meta } from '@storybook/blocks';
+import * as ProgressBarStories from './progress-bar.stories';
+
+<Meta of={ ProgressBarStories } name='Migration From 17.x' />
+
+# ProgressBar - migrate from v17 to v18
+----
+
+## Slot changes
+
+`end` <img src="https://img.shields.io/badge/WIP-00FFFF" />
```

Comment:
writes the doc as if current situation was the final one (cause it may be the case), so
- use remove badge like other component
- update description like other removed attribute